src/components/time/__tests__/ReadableTime.test.js

```diff
@@ -32,114 +36,53 @@ describe('components/time/ReadableTime', () => {
-            timestamp: msYesterday,
-        },
-        {
-            description: 'should render with distant past format "mm dd yy at hh:mm"',
-            timestamp: msBeginningOfCenturyTime,
-        },
-        {
-            description: 'should render two days ago with format "mm dd"',
-            timestamp: msTwoDaysAgo,
-        },
-        {
-            description: 'should render two days ago with format "mm dd at hh:mm" when we show the time',
-            timestamp: msTwoDaysAgo,
-            alwaysShowTime: true,
-        },
-        {
-            description: 'should render with weekday when we show weekday',
-            timestamp: msTwoDaysAgo,
-            showWeekday: true,
-        },
-        {
-            description: 'should render with format "Today at hh:mm" when there is a future time stamp',
-            timestamp: relativeThreshold * 2 + now,
-        },
-        {
-            description: 'should render with format "Today at hh:mm" when `allowFutureTimestamps` is false',
-            timestamp: ms1HourInFuture,
-            allowFutureTimestamps: false,
-        },
-        {
-            description:
-                'should render with format "in 30 minutes" when timestamp is within relative threshold (ahead)',
-            timestamp: withinRelativeThresholdAhead,
-        },
-        {
-            description:
-                'should render with format "30 minutes ago" when timestamp is within relative threshold (behind)',
-            timestamp: withinRelativeThresholdBehind,
-        },
-    ].forEach(
-        ({ description, timestamp, allowFutureTimestamps = true, alwaysShowTime = false, showWeekday = false }) => {
-            test(description, () => {
-                const wrapper = shallow(
-                    <ReadableTime
-                        intl={intl}
-                        allowFutureTimestamps={allowFutureTimestamps}
-                        alwaysShowTime={alwaysShowTime}
-                        relativeThreshold={oneHourInMs}
-                        showWeekday={showWeekday}
-                        timestamp={timestamp}
-                    />,
-                );
-
-                expect(wrapper).toMatchSnapshot();
-            });
+            expect(wrapper.children()).toMatchSnapshot();
+            wrapper.setProps({ uppercase: true });
+            expect(wrapper.children()).toMatchSnapshot('uppercase');
         },
     );
 
     test('should use default relative threshold if not provided', () => {
         const wrapper = shallow(<ReadableTime intl={intl} timestamp={withinRelativeThresholdAhead} />);
         expect(wrapper).toMatchSnapshot();
     });
+    test('should not uppercase locales that do not have uppercase grammar', () => {
+        const ruIntl = createIntl({ locale: 'ru', messages: {} });
+        // [React Intl] Missing locale data for locale: "ru". Using default locale: "en" as fallback.
```

Comment:
@ehoogerbeets https://github.com/formatjs/react-intl/blob/b207d805a9e5609983ac1b2ca8f9708030e12337/docs/Upgrade-Guide.md#migrate-to-using-native-intl-apis

not fully running this bc node doesn't ship w/ the full ICU package